px-map-behavior-layer-IMS.es6.js

```diff
@@ -238,12 +250,18 @@
       this.querySelector('#get-collection').generateRequest();
 
       //Bind to px-maps moveend to re-request the data with new bounds
+      //If layer is not going to be rendered at the current zoom level, don't load
       this.parentNode.elementInst.on({
         moveend: () => {
-          const bounds = this.parentNode.elementInst.getBounds();
-          const boundsArray = [bounds._southWest.lng, bounds._northEast.lng, bounds._southWest.lat, bounds._northEast.lat];
+          const layerStartingZoomValue = this.visibilityZoomLevels.substring(0,this.visibilityZoomLevels.indexOf("+"));
```

Comment:
While this is compact, it is not very readable. 
I would move the string parsing to a different function both to make it readable and so that we can expand it to deal with other cases later such as `10-14` or `11,12`